src/main/java/net/minecraftforge/event/entity/living/LivingSetAttackTargetEvent.java

```diff
@@ -27,16 +28,48 @@
  **/
 public class LivingSetAttackTargetEvent extends LivingEvent
 {
-
-    private final LivingEntity target;
+    private final ILivingTargetType targetType;
+    private final LivingEntity originalTarget;
+    
     public LivingSetAttackTargetEvent(LivingEntity entity, LivingEntity target)
```

Comment:
Deprecate, mark for removal.

Reply:
Fixed: d8371baa7a87a75a24e43080473f41f7cb0d75ba